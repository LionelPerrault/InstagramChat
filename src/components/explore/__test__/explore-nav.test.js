import React from 'react'
import { create } from 'react-test-renderer'
import ExploreNav from '../explore-nav'

describe('ExploreNav Component', () => {
  it('should match snapshot', () => {
    const tree = create(<ExploreNav url="/explore" />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
