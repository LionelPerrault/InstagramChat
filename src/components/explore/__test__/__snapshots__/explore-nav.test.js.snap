// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExploreNav Component should match snapshot 1`] = `
<ul>
  <li>
    <NavLink
      activeClassName="exp_nav_active"
      className="exp_nav_link"
      exact={true}
      to="/explore"
    >
      Users
    </NavLink>
  </li>
  <li>
    <NavLink
      activeClassName="exp_nav_active"
      className="exp_nav_link"
      to="/explore/explore-photos"
    >
      Photos
    </NavLink>
  </li>
  <li>
    <NavLink
      activeClassName="exp_nav_active"
      className="exp_nav_link"
      to="/explore/explore-groups"
    >
      Groups
    </NavLink>
  </li>
</ul>
`;
